libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1917,6 +2023,154 @@ void AP_OSD_Screen::draw_esc_amps(uint8_t x, uint8_t y)
 }
 #endif
 
+#if AP_OSD_CRSF_PANELS_ENABLED
+bool AP_OSD_Screen::is_btfl_fonts()
+{
+    const AP_MSP *p_msp = AP::msp();
+    return (p_msp && p_msp->is_option_enabled(AP_MSP::Option::DISPLAYPORT_BTFL_SYMBOLS));
+}
+
+void AP_OSD_Screen::draw_crsf_tx_power(uint8_t x, uint8_t y)
+{
+    const int16_t tx_power = AP::crsf()->get_link_status().tx_power;
+    bool btfl = is_btfl_fonts();
+    if (tx_power > 0)
+    {
+        if(tx_power < 1000) {
+            if(btfl) {
+                backend->write(x, y, false, "%3dMW", tx_power);
```

Comment:
should me mW not MW, as its milliWatts not MegaWatts

Reply:
I have made a quick change to use the 0x0C character instead of "M", and this is how it looks like:

![IMG_8101](https://github.com/ArduPilot/ardupilot/assets/9812730/1dd2ca53-a2ce-424f-8e98-d7fec01ec2b6)

Acceptable?